pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -809,6 +809,16 @@ func (l *LibvirtDomainManager) migrateHelper(vmi *v1.VirtualMachineInstance, opt
 	return nil
 }
 
+// prepareDomainForMigration perform necessary operation
+// on the source domain just before migration
+func prepareDomainForMigration(virtConn cli.Connection, domain cli.VirDomain) error {
+	if err := hotUnplugHostDevices(virtConn, domain); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Why don't you simply `return hotUnplugHostDevices(virtConn, domain)` ?

Reply:
Well.. no particular reason at least until there will be other operations
I will change it to what you suggests

Reply:
Well I feel ignored.

Reply:
??

Reply:
Sorry, replied on a cached version of this.

I tried to delete hit, but too late, the cat was out of the bag.

Sorry again @ormergi .